channel-email/src/main/java/com/synopsys/integration/alert/channel/email/database/accessor/EmailGlobalConfigAccessor.java

```diff
@@ -93,6 +94,9 @@ public EmailGlobalConfigModel createConfiguration(EmailGlobalConfigModel configu
     public EmailGlobalConfigModel updateConfiguration(UUID configurationId, EmailGlobalConfigModel configuration) throws AlertConfigurationException {
         EmailConfigurationEntity configurationEntity = emailConfigurationRepository.findById(configurationId)
             .orElseThrow(() -> new AlertConfigurationException(String.format("Config with id '%s' did not exist", configurationId.toString())));
+        if (BooleanUtils.toBoolean(configuration.getIsSmtpPasswordSet())) {
+            configuration.setSmtpPassword(configurationEntity.getAuthPassword());
+        }
```

Comment:
Need to add check to see if password exists in new body, use that new value instead